service/src/test/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleServiceTest.java

```diff
@@ -73,8 +91,11 @@ public void shouldMakeCorrectCallForFees(Boolean toggleState) {
         givenToggle(toggleState);
 
```

Comment:
Test above (shouldMakeCorrectCallForCtscReport) doesn't validate the submitted request, intended?